scanner/src/test/kotlin/scanners/scancode/ScanCodeResultParserTest.kt

```diff
@@ -41,28 +41,6 @@ import org.ossreviewtoolkit.spdx.SpdxConstants
 
 @Suppress("LargeClass")
 class ScanCodeResultParserTest : WordSpec({
-    "ScanCode 2 results" should {
```

Comment:
I believe we shouldn't remove these completely, but simply test other properties of a ScanSummary.

Reply:
Done for you in https://github.com/oss-review-toolkit/ort/pull/4233, so you just need to rebase after that got merged.